src/applications/vaos/utils/calendar.js

```diff
@@ -113,3 +116,96 @@
   });
   return map;
 }
+
+export function formatDescription(description, location = '') {
+  if (!description || !description.text) {
+    return 'DESCRIPTION:';
+  }
+
+  const chunked = [];
+  if (typeof description === 'object') {
+    let text = `DESCRIPTION:${description.text}`;
+    text = text.replace(/\r/g, '').replace(/\n/g, '\\n');
+
+    while (text.length > ICS_LINE_LIMIT) {
+      chunked.push(`${text}\\n`.substring(0, ICS_LINE_LIMIT));
+      text = text.substring(ICS_LINE_LIMIT);
+    }
+
+    // Add last line of description text
+    if (text) {
+      chunked.push(text);
+    }
+
+    if (description.providerName) {
+      chunked.push(`\\n\\n${description.providerName}`);
+    }
+
+    if (location) {
+      const loc = description.providerName
+        ? `\\n${location}`
+        : `\\n\\n${location}`;
+      const index = loc.indexOf(',');
+
+      if (index !== -1) {
+        chunked.push(`${loc.substring(0, index)}\\n`);
+        chunked.push(`${loc.substring(index + 1).trimStart()}\\n`);
+      } else {
+        chunked.push(`${loc}\\n`);
+      }
+    }
+
+    const phone = description.phone?.replace(/\r/g, '').replace(/\n/g, '\\n');
+    if (phone && phone !== 'undefined') {
+      chunked.push(`${phone}\\n`);
+    }
+
+    if (description.additionalText) {
+      description.additionalText.forEach(val => {
+        let line = `\\n${val}`;
+        while (line.length > ICS_LINE_LIMIT) {
+          chunked.push(`${line}\\n`.substring(0, ICS_LINE_LIMIT));
+          line = line.substring(ICS_LINE_LIMIT);
+        }
+        chunked.push(`${line}\\n`);
+      });
+    }
+  }
+
+  return chunked.join('\r\n\t').replace(/,/g, '\\,');
```

Comment:
Not sure how to fix this since the documentation says that using a regx with the global option should suffice. This code generates an .ics file which has to be in a specific format. Please advise.